aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1780,6 +1780,9 @@ class QConvoneDNN final {
         attr,
         ".")
     }
+
+    output_scale = 1.0 / output_scale;
```

Comment:
should probably rename this to inv_output_scale

Reply:
Thanks and changed.